app/src/main/res/layout/selection_interaction_item.xml

```diff
@@ -51,6 +54,9 @@
       app:allOptionsItemInputType="@{viewModel.getSelectionItemInputType()}"
       app:data="@{viewModel.choiceItems}"
       app:entityId="@{viewModel.entityId}"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
```

Comment:
This constraint should be `top_to_bottomOf` -> `TextView`

Reply:
Done.